src/backend/auth/admin_access_endpoints.py

```
@@ -0,0 +1,243 @@
+    org_role: Literal["org_admin", "org_member"]
+
+
+@admin_access_router.post("/orgs/{org_id}/owner")
+def set_owner(org_id: str, body: _SetOwner, request: Request,
+              admin: dict = Depends(require_admin)):
+    _require_active_user(body.user_id)  # no inactive owner
+    try:
+        _orgs.set_org_owner(org_id, body.user_id, body.org_role)
+    except (psycopg.errors.ForeignKeyViolation, psycopg.errors.InvalidTextRepresentation, ValueError):
+        # Bad UUID / unknown org-or-user / non-team org is a clean 400, not a 500.
+        raise HTTPException(400, "Unknown org/user or non-team org")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)